AniHyou/Screens/MediaList/ViewModels/MediaListViewModel.swift

```diff
@@ -87,9 +87,9 @@ class MediaListViewModel: ObservableObject {
     
     @Published var isLoading = false
     
-    func updateEntryProgress(entryId: Int, progress: Int) {
+    func updateEntryProgress(entryId: Int, progress: Int, status: GraphQLEnum<MediaListStatus>) {
```

Comment:
Can you make the status param nullable and change the type to MediaListStatus? So it's not sent if we don't really want to update it. Take a look at the [updateEntry](https://github.com/axiel7/AniHyou/blob/9e4256a949406558817dbeca8a695a99755ee9a3/AniHyou/Screens/MediaDetails/Edit/ViewModels/MediaListEditViewModel.swift#L63) function

Reply:
yep. I was trying to figure out how to do it. I'll give it another crack